jaypeasee/national-parkfinder
# Tdd/park info

## Is this a fix or a feature?
feature/testing

## What is the change?
Unit tests the following components:
* `ParkInfo`
* `About`
* `Location`
* `Contact`
*  `SavedCards`

## Where should the reviewer start?
Review the test files for each of the above components.

## How should this be tested?
1. See if there are sufficients tests for each component
2. run tests to confirm they are passing with:  `npm test src/<folder name>/<component name>.test.js`

## Relevant Tickets
* Closes #25
* Closes #18
* Closes #27
* Closes #45
* Closes #46

@saraho1123
@jaypeasee
@baileydunning
